sound/soc/sof/intel/hda-bus.c

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2018 Intel Corporation. All rights reserved.
+ *
+ * Authors: Jeeja KP <jeeja.kp@intel.com>
+ *          Keyon Jie <yang.jie@linux.intel.com>
+ */
+
+#include <sound/hdaudio.h>
+
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA)
```

Comment:
Isn't this implied by Kconfig option

Reply:
are you meaning that IS_ENABLED(CONFIG_SND_SOC_SOF_HDA) is true always?
the answer is no for that, it is tristate, we can choose them or not in menuconfig.